.prettierignore

```diff
@@ -1,7 +1,11 @@
 #! this file should only be used as a last resort to exclude files from pettier
-# you should always prefer to use // prettier-ignore to exluce individual lines
+# you should always prefer to use // prettier-ignore to exclude individual lines
 
 # we ignore the service token file from pettier so that we can one-line all of
 # the service tokens
 # this makes it much more readable as each service token has its ownn line
 src/app/services/baw-api/ServiceTokens.ts
+
+# we ignore .html files because prettier doesn't play nice with compiler
+# control flow (e.g. @if, @else, and @for)
```

Comment:
open issue on prettier? cross link?

Reply:
Looks like there is a `--parser angular` option that I should be using.

And it formats angulars new control flow syntax correctly

Reply:
Fixed in: [`6a42613` (#2270)](https://github.com/QutEcoacoustics/workbench-client/pull/2270/commits/6a426138fe3cce7650554621412d1f506723c8bb)